### DISCUSSION TITLE
using yamlip for 01 planning

### DISCUSSION THREAD

**Discussion by zjm-666** [yalmip/YALMIP#1349] [Optimization modelling]
I am using yamlip to solve a binary programming, I don't understand why I met this error, please help me to look at it, thank you very much! This is my code!
[code_matlab.txt](https://github.com/yalmip/YALMIP/files/13487475/code_matlab.txt)

**Comment by johanlofberg** [comment]
We don't know which error you have met so an impossible question.

Also, you cannot have read the tutorials, as you have some weird use of a command _double_ when setting up the constraint?
https://yalmip.github.io/tutorial/basics/

**Reply by johanlofberg** [reply]
also, conv2 is not overloaded on sdpvars, so you will have to implement that operation manually

**Reply by zjm-666** [reply]
Thank you for your reply, I will study again！
